src/code-editor/index.tsx

```diff
@@ -105,10 +90,22 @@ export default function CodeEditor(props: CodeEditorProps) {
 
   const errorsTabRef = useRef<HTMLButtonElement>(null);
   const warningsTabRef = useRef<HTMLButtonElement>(null);
-
+  useEffect(() => {
+    const elem = codeEditorRef.current;
+    if (!ace || !elem) {
+      return;
+    }
+    const config = getDefaultConfig();
+    setEditor(
+      ace.edit(elem, {
+        ...config,
+        theme: getAceTheme(getDefaultTheme(elem)),
+      })
+    );
+  }, [ace, props.loading]);
```

Comment:
Just a small change to finalise the PR, I had to add the `props.loading` as a dependency because the codeEditorRef is not initiated when the `props.loading` is false as the div holding the codeEditorRef as a ref is conditionally rendered based on that loading value.